internal/cluster/clusterworkflow/activity_create_node_pool.go

```diff
@@ -161,6 +167,44 @@ func (a CreateNodePoolActivity) Execute(ctx context.Context, input CreateNodePoo
 			}
 		}
 
+		var amiImageSize int
```

Comment:
The term `amiImage` is a bit redundant, given AMI stands for Amazon Machine Image. `amiSize` or just `imageSize` would sound better IMHO.

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/compare/0e30f5dbe8ec96b1e168c848aadfd65cfc408226..4a93f8bc4925e894ff035584174682ce66d91d93)
[in](https://github.com/banzaicloud/pipeline/compare/4a93f8bc4925e894ff035584174682ce66d91d93..57884283dad426463fbd7975ddce43c4b9ee3f25)
[iterations](https://github.com/banzaicloud/pipeline/compare/57884283dad426463fbd7975ddce43c4b9ee3f25..64505f3f784b8c3a3ea26719817b881744a571f6).